public/app/plugins/datasource/cloudwatch/composable_dataquery.cue

```diff
@@ -0,0 +1,137 @@
+// Copyright 2023 Grafana Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grafanaplugin
+
+import (
+	common "github.com/grafana/grafana/packages/grafana-schema/src/common"
+	"github.com/grafana/grafana/pkg/plugins/pfs"
+)
+
+// This file (with its sibling .cue files) implements pfs.GrafanaPlugin
+pfs.GrafanaPlugin
+
+composableKinds: DataQuery: {
+	maturity: "merged"
+
+	lineage: {
+		seqs: [
+			{
+				schemas: [
+					{
+						common.DataQuery
+						//#CloudWatchMetricsQuery | #CloudWatchLogsQuery
```

Comment:
@sdboyer this doesn't seem to work, when I uncomment this I get an error `PluginTSTypesJenny: generating TS for child elements of schema failed: composableKinds.DataQuery.lineage.seqs.0.schemas.0: non-concrete value struct for input "cloudwatch"`.